.github/workflows/hardhat-network-tracing-scheduled.yml

```diff
@@ -0,0 +1,44 @@
+name: Hardhat Network Tracing Capabilities, scheduled runs for all solc verisons
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
This is midnight, right? Let's add a comment here, just because cron syntax always gives me pause.